roles/openshift_metrics/tasks/start_metrics.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- command: >
+    {{openshift.common.client_binary}} 
+    --config={{mktemp.stdout}}/admin.kubeconfig 
+    get rc 
+    -l metrics-infra=hawkular-cassandra 
+    -o name 
+    -n {{openshift_metrics_project}}
+  register: metrics_cassandra_rc
+  changed_when: no
+
+- name: Start Hawkular Cassandra
+  include: scale.yaml
+  vars:
+    desired: 1
```

Comment:
we expect statefulesets (petsets) to be the path forward which will work like a traditional replicaset but for now we are limited to a single RC per Cassandra Node.  I don't see any other way to control this for now.  It just means in future when statefulsets come into play we will have to modify the ansible tasks.  I would expect this anyway since we would move from RC->SS